stable/aws-calico/README.md

```diff
@@ -52,6 +52,7 @@ The following table lists the configurable parameters for this chart and their d
 | `fullnameOverride`      | Override the fullname of the chart                      | `calico`                        |
 | `serviceAccount.name`   | The name of the ServiceAccount to use                   | `nil`                           |
 | `serviceAccount.create` | Specifies whether a ServiceAccount should be created    | `true`                          |
+| `podSecurityPolicy.create` | Specifies whether podSecurityPolicy and related rbac objects should be created    | `false`                          |
```

Comment:
Note that @casparrolfe cleaned up the formatting of this markdown table in #233. So you might want to do the same in a future commit. Merging this now.